### PR TITLE
Media form field bootstrapped with static preview

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #10800 
#### Summary of Changes

Restore the static preview as an alternative to the pop over

Just a note here:
#### Testing Instructions

Apply this PR
edit administrator/components/com_content/models/forms/article.xml and replace the intro image field with:

``` XML
        <field
            name="image_intro"
            type="media"
            label="COM_CONTENT_FIELD_INTRO_LABEL"
            description="COM_CONTENT_FIELD_INTRO_DESC"
            preview="true"
            preview_width="300"
            preview_height="300"
        />
```

edit an article and observe the changes!

PS for maintainers this also includes the #10788 which is already RTC

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on c9f424642fc08aae8c3ee4cf5eedb30e281ae43e<br /><br />Assuming I tested this correctly this fails

With current staging I modified the intro_image field as described
Selected an image and observed it was displayed in a fixed preview
Applied patch
Observed the image was still displayed in a fixed preview but formatted differently
Selected a different image and that image was previewed AND the previous image. 
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10889">issues.joomla.org/joomla-cms/10889</a>.</sub>

**Comment by dgt41** [comment]
Still missing some proper formatting (css)

**Comment by dgt41** [comment]
By the way the old styling is also way off:
screen shot 2016-06-21 at 14 39 49

with this PR and no styling whatsoever i thing we are way better:
  
screen shot 2016-06-21 at 14 47 25

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 02a728bfe3ac3f2786716190fb495f84fafe9c80<br /><br />After applying the path the following tabs are all displayed on the same tab
Images and links
Associations
Options
Configure Edit Screen
Permissions<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10889">issues.joomla.org/joomla-cms/10889</a>.</sub>

**Comment by piotr-cz** [comment]
What do you think about showing the preview under field controls?
Usually we have label on the left side and controls on the right.

In the preview you've added when I'd like to change the image, controls are not on the same line and I have to look where the image is ending, eventually scroll down and then click.

**Comment by dgt41** [comment]
@piotr-cz makes sense, will do that

**Comment by dgt41** [comment]
#### Preview:
Fixed:
screen shot 2016-06-25 at 10 40 53

As popup:
screen shot 2016-06-25 at 10 41 38

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on eff5df51de99d0a44b611679e3bae7e8744e37a8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10889">issues.joomla.org/joomla-cms/10889</a>.</sub>

**Comment by Ruchiranga** [comment]
I have tested this item :white_check_mark: successfully on eff5df51de99d0a44b611679e3bae7e8744e37a8<br /><br />Screen shot

https://ibin.co/2mC88rU83NKs.png<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10889">issues.joomla.org/joomla-cms/10889</a>.</sub>

**Comment by zero-24** [comment]
RTC.

@dgt41 should we close https://github.com/joomla/joomla-cms/pull/10788 than? As this PR includes the code as per your comment?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10889">issues.joomla.org/joomla-cms/10889</a>.</sub>

**Comment by dgt41** [comment]
@zero-24 sure, I will do that. In any case we have a link if needed 😀

**Comment by dgt41** [comment]
Re synced!

**Comment by wilsonge** [comment]
So as #10788 was closed in favour of this (reading you comments in the main PR description) exactly what is the status here?

File: administrator/components/com_media/views/images/tmpl/default.php [hunk 1]
```diff
@@ -34,6 +37,26 @@
...
 	"
 );
+
+/**
+ * Mootools compatibility
+ *
+ * There is an extra option passed in the url for the iframe &ismoo=0 for the bootstrap fields.
+ * By default the value will be 1 or defaults to mootools behaviour
+ *
+ * This should be removed when mootools won't be shipped by Joomla.
```
Comment (wilsonge) [inline administrator/components/com_media/views/images/tmpl/default.php]: Please add a deprecated tag here with this

Comment (dgt41) [inline administrator/components/com_media/views/images/tmpl/default.php]: Deprecated 4.0 ?
Needs to be added also for user field which is already merged

**Comment by dgt41** [comment]
It was RTC but had merge conflicts which I just resolved. I don't know if new tests are needed

File: administrator/components/com_media/views/images/tmpl/default.php (hunk 1 above)
Comment (wilsonge) [inline administrator/components/com_media/views/images/tmpl/default.php]: Well as mootools is deprecate. We may as well deprecate the mootools support parameter ;)

**Pull request merged**